### ISSUE TITLE
Level: Big Brother

### ISSUE THREAD

**Issue by andybalaam** [andybalaam/rabbit-escape#531]
See PR #505 .

Logging as an issue so we don't miss it.

**Comment by GamingInky** [comment]
I have a new solution for this level: bridge&(6,8);4;(6,8);2;climb;4;(6,7);;dig&(3,10)&bridge&(1,13);11;(2,13);6;(3,12);7;(10,13);15;dig;(7,0);bridge;3;dig;(8,10)&bridge;10;bash;2;(10,10);(7,13);bridge;(9,13);8;(8,12);9

**Comment by GamingInky** [comment]
The new level is
```
:name=Big Brother
:description=Big Brother sees all.
:author_name=tttppp
:author_url=https://github.com/tttppp
:solution.1.code=82Gh(8~hw8~l82Gdd33""((88R((R(8G8"CZ^~8Q8w3hd~!((2gu((8d(8Gd88!GQc88(8cG82G8u"(82G8whc88~hdu2"8Rcd2G88uhd8OG228uR8gGQh8dd!~68c8h(83lh28((h2R6(G2R8hG!d28Ghh(8((hR((2"
:solution.2.code=N;^F4ki+i+vNGLTkU+44FT+^,*7cc/*<Fov+Ni+L^Fuv74TGLFN++uN+kN+47T^+vi^,+Lc,kiu+ /TTTvNvLFT+F7k+NG+L^iTiNTLoG,+Fu7/FLv<FTv^7c4TN^++u<w+,+eFF47<vvv7Tc+/N+v+k+/Fo+oiKGF3vT^+N>TvkNT++NwT++
:num_rabbits=0
:num_to_save=3
:rabbit_delay=4
:bash=2
:bridge=9
:climb=1
:dig=9
 #*       # 
 ########## 
            
            
            
    MMMM    
    *  M    
    MM M    
MM  M  M  MM
M  #M#MM#  M
#    O     #
#####MM#####
#*   M     #
#\  /#\  */#
##### ######
:*=t{index:1}
:*=r{index:2}
:*=r{index:3}
:*=j{index:4}
```

**Comment by tttppp** [comment]
I like the extra solution - careful timing required to pull it off.

**Comment by andybalaam** [comment]
Yep I like it too, and merged it in 3c190f8b919811f2c21232e7ae9460f303801684

**Comment by andybalaam** [comment]
Included in Rabbots.
